guide/how-it-works/lightning-service-providers.md

```diff
@@ -0,0 +1,310 @@
+
+As the users recovery phrase is required to recover the channel states stored by the LSP, this makes the recovery phrase wallet-dependent. Users will be unable to recover their funds in wallet that was not the one who generated it, making things less [interoperable]({{ '/guide/designing-products/wallet-interoperability/' | relative_url }}).
+
+Users should have the option to backup payment channel states [manually]({{ '/guide/onboarding/backing-up-a-wallet/manual-backup/' | relative_url }}) or [automatically]({{ '/guide/onboarding/backing-up-a-wallet/cloud-backup/' | relative_url }}).
+
+[ACINQ](https://medium.com/@ACINQ/phoenix-wallet-part-3-backup-f63a9470d4e7) is an example of a LSP using a backup and recovery service for its wallet Phoenix. [Backpack](https://github.com/synonymdev/backpack-client) is an open-source client enabling this service.
+
+</div>
+
+## Channel opens  
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/lightning-service-providers/channel-opens.jpg"
+   retina = "/assets/images/guide/how-it-works/lightning-service-providers/channel-opens@2x.jpg"
+   alt-text = "Image showing lightning channels represented like an abacus.."
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+To use a lightning wallet, users first need a payment channel. Manually opening a payment channel involves finding a suitable peer, having on-chain bitcoin in your wallet to put in the channel, and broadcasting a transaction. Once this channel is opened, it needs to be managed.
+
+All of this can be complex for users to deal with. A channel opening service automatically opens a channel to the LSP for the user when necessary.
+
+Users should always have the option to manually open channels, as to not completely rely on a LSP.
+
+Below, we cover the different ways in which a channel open service can be offered, depending on user needs and the trade-offs they are willing to make.
+
+</div>
+
+### Inbound liquidity
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/lightning-service-providers/inbound-liquidity.jpg"
+   retina = "/assets/images/guide/how-it-works/lightning-service-providers/inbound-liquidity@2x.jpg"
+   alt-text = "Drop of water representing liquidity."
+   width = 140
+   height = 140
+   layout = "float-right"
+%}
+
+Lightning wallets can not receive payments without inbound liquidity in a channel. This is primarily the case with new wallets, but may also occur when channels are imbalanced on funded wallets.
+
+Offering inbound liquidity with a channel open service addresses this situation. When a user receives a payment without enough inbound liquidity, a channel is automatically opened. The network fee, and sometimes an additional service fee, for opening the channel is typically deducted from the payment amount.
```

Comment:
> inbound liquidity

This would be a great thing to add to glossary I think as we don't explain it anywhere in the guide. Just leaving it here for the reference.

Reply:
We have an issue up to create a whole page on this topic https://github.com/BitcoinDesign/Guide/issues/685,  but agree more lightning terminology needs to be defined in the glossary.